lib/process-services-cloud/src/lib/form/components/cloud-form-rendering.service.ts

```diff
@@ -45,7 +46,8 @@ export class CloudFormRenderingService extends FormRenderingService {
             [FormFieldTypes.RADIO_BUTTONS]: () => RadioButtonsCloudWidgetComponent,
             [FormFieldTypes.ALFRESCO_FILE_VIEWER]: () => FileViewerWidgetComponent,
             [FormFieldTypes.DISPLAY_RICH_TEXT]: () => DisplayRichTextWidgetComponent,
-            [FormFieldTypes.DATA_TABLE]: () => DataTableWidgetComponent
+            [FormFieldTypes.DATA_TABLE]: () => DataTableWidgetComponent,
+            [FormFieldTypes.DISPLAY_EXTERNAL_PROPERTY]: () => DisplayExternalPropertyWidgetComponent
```

Comment:
at this point why are u not adding the component on core? I mean here we should suppose to override component for a basic type. it sounds very strange that u add the type on core but only process will implement it. What would other libs render when using the new type?

Reply:
as we discussed, raised issue for investigate it -> https://alfresco.atlassian.net/browse/AAE-20917